raspap.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
+// Default admin username and password:
 $config = array(
   'admin_user' => 'admin',
   'admin_pass' => '$2y$10$YKIyWAmnQLtiJAy6QgHQ.eCpY4m.HCEbiHaTgN6.acNC6bDElzt.i'
 );
 
-if(file_exists(RASPI_CONFIG.'/raspap.auth')) {
-    if ( $auth_details = fopen(RASPI_CONFIG.'/raspap.auth', 'r') ) {
+// Can be overridden by what's in this file, if it exists:
+if(file_exists(RASPI_ADMIN_DETAILS)) {
```

Comment:
We don't seem to set a value for `RASPI_ADMIN_DETAILS` ...

Also, wondering if we shouldn't rename the variable to something like RASPI_AUTH_FILE to be more specific on its contents.

Reply:
Oh, or ALLSKY_AUTH_FILE

Reply:
@linuxkidd, RASPI_ADMIN_DETAILS and all the other RASPI_* variables are defined in index.php (which includes almost all the files in the "includes" directory).  It's  `define('RASPI_ADMIN_DETAILS', RASPI_CONFIG . '/raspap.auth');`

I think we should  dump the term "RASPI" in variable names and use "ALLSKY" instead.  ALLSKY_AUTH_FILE would be good.   HOWEVER, let's wait to change setting and variable names until we do the unified json file.  Since that PR will require changes in almost every file, it'll be a good opportunity to rename things at the same time.

Sound reasonable?